src/client.cc

```diff
@@ -106,7 +111,9 @@ void Client::FlagShoppingState(
   client_state_->shop_url = url;
   client_state_->score = score;
 
-  client_state_->last_shop_time = helper::Time::Now();
+  auto now = base::Time::Now();
```

Comment:
Seems like `NowInSeconds` would be a good candidate for a member function, since we're doing it so much.

Reply:
👍 good spot, fixed